.github/workflows/release-pkg.nu

```diff
@@ -99,6 +99,7 @@ if $os in [$USE_UBUNTU, 'macos-latest'] {
 # Build for Windows without static-link-openssl feature
 # ----------------------------------------------------------------------------
 if $os in ['windows-latest'] {
+    # let-env CARGO_BUILD_TARGET = $target
```

Comment:
Is this commented-out line intentional?

Reply:
Yes, it is.

Reply:
What is it for?

Reply:
The original script works too good to be true, I'm just trying another way to specify the cargo build target to make sure it works. Of course, you can remove it.